### ISSUE TITLE
Typescript error: nuxt.config.ts build.babel doesn't have presets in TransformOptions

### ISSUE THREAD

**Issue by jackie-park** [nuxt/nuxt.js#5774]
### Version

[2.7.1](https://github.com/nuxt.js/releases/tag/2.7.1)

### Reproduction link

[https://github.com/nuxt/nuxt.js/blob/dev/packages/config/types/build.d.ts](https://github.com/nuxt/nuxt.js/blob/dev/packages/config/types/build.d.ts)

### Steps to reproduce

Typescript error: nuxt.config.ts build.babel doesn't have presets in TransformOptions

### What is expected ?

export interface NuxtConfigurationBuild {
  babel?: TransformOptions
}

I expected TransformOptions should have presets

### What is actually happening?

const nuxtConfig: NuxtConfiguration = {
build: {
  presets <====
}
}
presets which makes typescript error

<div align="right"><sub><em>This bug report is available on <a href="https://cmty.app/nuxt">Nuxt</a> community (<a href="https://cmty.app/nuxt/nuxt.js/issues/c9245">#c9245</a>)</em></sub></div>

**Comment by kevinmarrec** [comment]
@jackie-park 

`TransformOptions` interface has `presets` property : https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/babel__core/index.d.ts#L254

You seem to do

```
build : {
  presets:
}
```
instead of 
```
build: {
  babel: {
    presets:
  }
}
```

![Screenshot from 2019-05-21 12-02-44](https://user-images.githubusercontent.com/25272043/58087203-63370c00-7bc0-11e9-9dbf-7202712c09a4.png)

**Comment by jackie-park** [comment]
It was no problem with 2.6.x and works well in 2.7.x when I remove NuxtConfiguration type.
Theres's a test below in nuxt.js project.
https://github.com/nuxt/nuxt.js/blob/d811ef7fea697f43a75ef4fb93f5a9a706ec5d6d/packages/config/test/options.test.js#L212

**Comment by kevinmarrec** [comment]
@jackie-park I still don't get what's wrong, please provide exact configuration snippet that throw an error and please also provide this error.

**Comment by jackie-park** [comment]
![image](https://user-images.githubusercontent.com/38970917/58228062-59500d00-7d68-11e9-8cca-db472ab94d28.png)

**Comment by kevinmarrec** [comment]
@jackie-park Alright, this PR https://github.com/nuxt/nuxt.js/pull/5785 fixes the issue (when `presets` function is used) and will be available in next release.

In the meantime I think you should be able to put a `// @ts-ignore` comment above the `presets` so it doesn't throw an error.

**Comment by jackie-park** [comment]
@kevinmarrec Thank you 👍

**Comment by kevinmarrec** [comment]
Fix released in [2.8/0](https://github.com/nuxt/nuxt.js/releases/tag/v2.8.0)
